endpoints/api/test/test_user.py

```diff
@@ -40,3 +41,106 @@ def test_user_metadata_update(client):
 
             # The location field should be unchanged.
             assert user.get("location") == location
+
+
+@pytest.mark.parametrize(
+    "user_count, expected_code, feature_mailing, feature_user_initialize, metadata",
+    [
+        # Feature flag false
+        (
+            0,
+            400,
+            True,
+            False,
+            {
+                "username": "someone",
+                "password": "password",
+                "email": "someone@somewhere.com",
+            },
+        ),
+        # Non-empty database fails
+        (
+            1,
+            400,
+            True,
+            True,
+            {
+                "username": "someone",
+                "password": "password",
+                "email": "someone@somewhere.com",
+            },
+        ),
+        # Empty database with mailing succeeds
+        (
+            0,
+            200,
+            True,
+            True,
+            {
+                "username": "someone",
+                "password": "password",
+                "email": "someone@somewhere.com",
+            },
+        ),
+        # Empty database without mailing succeeds
+        (
+            0,
+            200,
+            False,
+            True,
+            {
+                "username": "someone",
+                "password": "password",
+            },
+        ),
+        # Empty database with mailing missing email fails
+        (
+            0,
+            400,
+            True,
+            True,
+            {
+                "username": "someone",
+                "password": "password",
+            },
+        ),
+        # Empty database with access token
+        (
+            0,
+            200,
+            False,
+            True,
+            {"username": "someone", "password": "password", "access_token": "true"},
```

Comment:
Not sure what case this is. Shouldn't be able to have a token without a user..

Reply:
This is testing success w/o mailing feature. Not sure what comment means.